changelog.d/20241205_211837_danyal.faheem_add_dev_attach_command.md

```diff
@@ -0,0 +1 @@
+- [Improvement] Attach container stdin when running tutor dev start <service> for debugging. (by @Danyal-Faheem)
```

Comment:
```suggestion
- [Bugfix] Fix breakpoint debugging by attaching container stdin when running tutor dev start <service> for a single <service>. (by @Danyal-Faheem)
```

I'd consider this a bugfix, since the docs have incorrectly stated that this worked for a while.